File: Level 071/homework/hw1.js
/* 2)for-ციკლის გამოყენებით იპოვეთ სიაში ყველაზე დიდი და ყველაზე პატარა რიცხვები */

let my_list = [12, 28, 30, 50, 70, 100]

let min = my_list[0]
let max = my_list[0]

for (let i = 1; i < my_list.length; i++) {
    if (my_list[i] < min) {
        min = my_list[i]
    }

    else if (my_list[i] > max) {
        max = my_list[i]
    }
}

console.log(min)
console.log(max)

File: Level 030/homework/hw1.py
# 2)  შექმენით ფუნქცია რომელსაც არგუმენტად გადაეცემა რიცხვებით სავსე სია ამ ფუნქციამ კი უნდა დააბრუნოს ამ სიის რიცხვების ჯამი

def fun(my_list):
    final_list = 0

    for i in my_list:
        final_list += i
    return final_list

print(fun(([201, 239, 12, 212])))